### PR TITLE
assignment statement mutator#43

### PR DESCRIPTION
**Pull request by manuc66**
Implements assignment statement mutator #43

### CODE REVIEW

**Comment by richardwerkman** [comment]
Hey Emmanuel, thank you for this pull request too. The branch now contains conflicts with master.

**Comment by manuc66** [comment]
Hi @richardwerkman the conflit is now solved and I removed the commented section

File: src/Stryker.Core/README.md [hunk 1]
```diff
@@ -39,6 +40,19 @@ Stryker supports a variety of mutators, which are listed below. Do you have a su
...
+| Original | Mutated | 
+| ------------- | ------------- | 
+|`+= `	| `-= ` |
+|`-= `	| `+= ` |
+|`*= `	| `/= ` |
+|`/= `	| `*= ` |
+|`%= `	| `*= ` |
+|`<<=`  | `>>=` |
+|`>>=`  | `<<=` |
+|`&= `	| `\|= ` |
```
Comment (jnyrup) [inline src/Stryker.Core/README.md]: The mutator uses `SyntaxKind.ExclusiveOrAssignmentExpression` which is `^=`.
`|=` is `SyntaxKind.OrAssignmentExpression`.

**Comment by richardwerkman** [comment]
@jnyrup thanks for noticing!

**Pull request merged**